### PR TITLE
Fix-#1640 Added version argument to installation script

### PR DESCRIPTION
**Pull request by Tushar-kalsi**
## Related Issue
  - Info about Issue or bug

Closes: #1639

#### Describe the changes you've made
A clear and concise description of what you have done to successfully close your assigned issue. Any new files? or anything you feel to let us know!

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## Please let us know if any test cases are added
No test are added. 

Please describe the tests(if any). Provide instructions how its affecting the coverage.

#### Describe if there is any unusual behaviour of your code(Write `NA` if there isn't)
A clear and concise description of it.
1: Added version in url from where tar files are downloaded in functions.
2: Added version in docker pull command .
3: Updated documentation and way to install specific version .

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] New and existing unit tests pass locally with my changes.

## Screenshots (if any)

|        Original         |          Updated           |
|:-----------------------:|:--------------------------:|
| **original screenshot** | <b>updated screenshot </b> |

### CODE REVIEW

**Comment by Tushar-kalsi** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by Tushar-kalsi** [comment]
@slayerjain  can you check my PR , i am not able to pass DCO check .

**Comment by slayerjain** [comment]
@Tushar-kalsi you should ensure that all the commits in the branch are signed. Please refer to this - https://stackoverflow.com/questions/13043357/git-sign-off-previous-commits

The DCO only cares of the commits in this particular branch. ie https://github.com/keploy/keploy/pull/1644/commits

**Comment by Tushar-kalsi** [comment]
@slayerjain Done thanks !! DCO check passed .

**Comment by gouravkrosx** [comment]
@Tushar-kalsi Please resolve the merge conflicts. We have removed this `\$HOME/.keploy-config:/root/.keploy-config` mounting from the docker alias after refactoring.

**Comment by re-Tick** [comment]
@shivamsouravjha please review.

File: README.md [hunk 1]
```diff
@@ -121,9 +121,19 @@ On MacOS and Windows, additional tools are required for Keploy due to the lack o
...
+To install particular fuction of keploy 
+```shell
+curl -O https://raw.githubusercontent.com/keploy/keploy/main/keploy.sh && source keploy.sh installKeploy -version=$version
+```
 
+Example :
 ```shell
-curl -O https://raw.githubusercontent.com/keploy/keploy/main/keploy.sh && source keploy.sh
+curl -O https://raw.githubusercontent.com/keploy/keploy/main/keploy.sh && source keploy.sh installKeploy -version=v2.0.0-alpha43
 ```
```
Comment (shivamsouravjha) [inline README.md:138]: Hey Tushar, would it be possible to move out installKeploy back to script?

As that approach would be shorted to execute. and won't be function name dependent

**Comment by shivamsouravjha** [comment]
hey @Tushar-kalsi can you please address the comments and resolve the merge conflicts.

**Comment by shivamsouravjha** [comment]
closing the PR as issue is solved

**Pull request closed without merging**